### PR TITLE
[Merged by Bors] - chore: namespace and tidy MonCat/GroupCat adjunctions

### PR DESCRIPTION
**Pull request by kim-em**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
Looks good to me; nice clean-up!

Let me exercise my new-found power and try to
maintainer merge

**Comment by grunweg** [comment]
Let's try again:
maintainer merge

**Comment by grunweg** [comment]
Oops, that was twice. Sorry, I'm still learning :-)

File: Mathlib/Algebra/Category/GroupCat/Adjunctions.lean [hunk 1]
```diff
@@ -130,28 +130,20 @@ def adj : free ⊣ forget GroupCat.{u} :=
...
 def abelianize : GroupCat.{u} ⥤ CommGroupCat.{u} where
-  obj G :=
-    { α := Abelianization G
-      str := by infer_instance }
-  map f :=
-    Abelianization.lift
-      { toFun := fun x => Abelianization.of (f x)
-        map_one' := by simp
-        map_mul' := by simp }
+  obj G := CommGroupCat.of (Abelianization G)
```
[the remark this answers is not in the document]
Comment (fpvandoorn) [reply Mathlib/Algebra/Category/GroupCat/Adjunctions.lean:138]: LGTM

**Comment by fpvandoorn** [comment]
I believe you requested a change. Generally we do `maintainer merge` after these changes have been incorporated.

bors d+

File: Mathlib/Algebra/Category/MonCat/Adjunctions.lean [hunk 2]
```diff
@@ -62,23 +64,23 @@ def adjoinOneAdj : adjoinOne ⊣ forget₂ MonCat.{u} SemigroupCat.{u} :=
...
-#align free free
+#align free MonCat.free
 
 /-- The free-forgetful adjunction for monoids. -/
 def adj : free ⊣ forget MonCat.{u} :=
   Adjunction.mkOfHomEquiv
     { homEquiv := fun X G => FreeMonoid.lift.symm
       homEquiv_naturality_left_symm := fun _ _ => FreeMonoid.hom_eq (fun _ => rfl) }
-#align adj adj
+#align adj MonCat.adj
```
[the remark this answers is not in the document]
Comment (semorrison) [reply Mathlib/Algebra/Category/MonCat/Adjunctions.lean:83]: Done.

**Comment by semorrison** [comment]
bors merge

**Comment by grunweg** [comment]
> I believe you requested a change. Generally we do `maintainer merge` after these changes have been incorporated.
> 
> bors d+

Makes sense, thanks for letting me know!

**Pull request closed without merging**